scripts/performance/perf_processes.py

```diff
@@ -470,42 +521,74 @@ def __init__(self, *args, **kwargs):
             "sex": {"raw": "value1", "encoded": "123"},
             "height": {"raw": "value1", "encoded": "456"}
         })
+        self._max_cred_num = None
+        self._submitter_did = None
+        self._pool_handle = None
+        self._cred_offer_json = None
+        self._tails_writer = None
+
+    async def _upd_revoc_reg(self):
```

Comment:
So, GetRevocRegEntry request consists of GET and WRITE every time, right?
Should we mention this specific in docs, so that QA can calculate the expected load from the script appropriately?